src/ng-prettyjson.js

```diff
@@ -26,13 +26,18 @@ angular.module('ngPrettyJson', [])
       scope.settings = (scope.settings || {});
       scope.triggers = (scope.triggers || {});
       scope.edition = attrs.edition;
-      scope.aceEditor = window.ace !== undefined;    
+      scope.aceEditor = window.ace !== undefined;
+
+      scope.showEditButton = true;
+      if(scope.settings.showEditButton === false) {
+        scope.showEditButton = false;
+      }
```

Comment:
Above four lines can be narrowed down into

scope.showEditButton = _.isDefined(settings.showEditButton) ? settings.showEditButton : true;